pom.xml

```diff
@@ -53,9 +53,11 @@
     <changelist>-SNAPSHOT</changelist>
     <jenkins.version>2.60.3</jenkins.version>
     <java.level>8</java.level>
-    <scm-api.version>2.2.7</scm-api.version>
+    <scm-api.version>2.4.2-SNAPSHOT</scm-api.version>
```

Comment:
You can fix the CI errors. [Guide](https://github.com/jenkinsci/incrementals-tools/blob/master/README.md)

(Note that you will need to first bring https://github.com/jenkinsci/scm-api-plugin/pull/71 up to date with the `master` branch.)